backend/main.go

```diff
@@ -202,17 +226,41 @@ func main() {
 					if templateArg == "" {
 						log.Fatal().Msg("Template is required")
 					}
-
 					if len(emailList) == 0 {
 						log.Fatal().Msg("Email list is required for blasting email minimum 1 email")
 					}
 
+					htmlContent, err := os.ReadFile(templateArg)
+					if err != nil {
+						log.Fatal().Err(err).Msg("Failed to read template")
+					}
+
+					mailSender := NewMailSender(&MailConfiguration{
+						SmtpHostname: config.SmtpHostname,
+						SmtpPort:     config.SmtpPort,
+						SmtpFrom:     config.SmtpFrom,
+						SmtpPassword: config.SmtpPassword,
+					})
 					for _, email := range emailList {
+						mail := &Mail{
+							RecipientName:  email,
+							RecipientEmail: email,
+							Subject:        "TeknumConf - Attendee Waitlist",
+							PlainTextBody: fmt.Sprintf(`Hello, %s!
+					Thank you for participating on TeknumConf, due to the limited seating quota, you are on a waitlist.
+					Not to worry, you will receive an email from us regarding the seating in about 7 days.
+					Please do contact us if you didn't receive any email by then.`, email),
+							HtmlBody: string(htmlContent),
+						}
```

Comment:
3 ini jangan di hardcode, tapi ambil argument dari flags aja. Tapi, flags nya itu path to file, jadi lebih rapih, command ini harusnya buka 2 file (plain text body & html body), subject tetep string.

Flag nya juga jangan dibikin global diatas, tapi khusus di command ini aja. Jadi kurang lebih cara pakenya:

```sh
conf-backend blast-email --smtp.hostname=127.0.0.1 --smtp.port=587 --smtp.from=admin@localhost --smtp.password=supersecure --subject="Hello world" --plaintext-body /path/to/file --html-body /path/to/file
```

Reply:
noted.
btw, kepikiran juga, kalau list mail nya diambil dari file juga pye? dari csv misal?

username,usermail@mail.com

Reply:
Boleh kok